pkgs/tools/misc/heimdall/default.nix

```diff
@@ -51,17 +49,16 @@ mkDerivation rec {
       mkdir -p $out/{bin,share/doc/heimdall,lib/udev/rules.d}
       install -m755 -t $out/bin                bin/*
       install -m644 -t $out/lib/udev/rules.d   ../heimdall/60-heimdall.rules
-      install -m644 ../Linux/README   $out/share/doc/heimdall/README.linux
-      install -m644 ../OSX/README.txt $out/share/doc/heimdall/README.osx
+      install -m644 ../README.md               $out/share/doc/heimdall/README.md
+      install -m644 ../doc/*                   $out/share/doc/heimdall/
     '';
 
   meta = with lib; {
     broken = stdenv.hostPlatform.isDarwin;
-    homepage = "http://www.glassechidna.com.au/products/heimdall/";
     description = "Cross-platform tool suite to flash firmware onto Samsung Galaxy S devices";
     license = licenses.mit;
     maintainers = with maintainers; [ peterhoeg ];
```

Comment:
The original maintainer (probably) last touched this package 8 years ago. Given that I consider this more of a removal and readd in one step, should I add myself here?

Reply:
Please do.